sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -449,6 +451,11 @@ public virtual async Task StopTestRecordingAsync()
             if (Recording != null)
             {
                 await Recording.DisposeAsync(save);
+
+                if (Mode == RecordedTestMode.Record && save)
```

Comment:
Hmm so this will prompt everyone immediately once this is merged but before their assets are migrated. Should we make this dependent on whether the assets are migrated?

Reply:
There's a check in `AssertTestProxyToolIsInstalled` that skips the prompt if `AssetsJsonPath` is equal to null. I believe this does what you're suggesting.